java-manta-client-kryo-serialization/src/test/java/com/joyent/manta/serialization/CipherSerializerTest.java

```diff
@@ -50,10 +50,12 @@ public void canSerializeAesGcm128() throws Exception {
         canSerializeCipher(AesGcmCipherDetails.INSTANCE_128_BIT);
     }
 
+    @Test(groups = {"strong-crypto"})
```

Comment:
I'm a little worried about the implication that "128" isn't "strong" crypto.  Maybe "unlimited" to align with what the policy files are called?